api/src/main/java/org/cardanofoundation/rosetta/common/util/RosettaConstants.java

```diff
@@ -150,6 +150,8 @@ public enum RosettaErrorType {
     OUTSIDE_VALIDITY_INTERVAL_UTXO(
         "Error when sending the transaction - OutsideValidityIntervalUTxO", 4037),
     CONFIG_NOT_FOUND("Environment configurations needed to run server were not found", 4038),
+    LATEST_BLOCK_NOT_FOUND("Сan't get the latest block. The node has not yet been initialized",
```

Comment:
If we can't find any block (because the node has not started yet), then it's an error `GENESIS_BLOCK_NOT_FOUND`. Else, it's just some block that could not be found.
I believe that we could omit a new error creation.